app/assets/javascripts/views/postChatSurvey/PostChatSurveyDigitalAssistant.js

```diff
@@ -0,0 +1,111 @@
+                      </div>
+                      <div class="govuk-radios__item">
+                        <input class="govuk-radios__input" id="q3--2" name="q3-" type="radio" value="Webchat with an HMRC adviser">
+                        <label class="govuk-label govuk-radios__label" for="q3--2">Webchat with an HMRC adviser</label>
+                      </div>
+                      <div class="govuk-radios__item">
+                        <input class="govuk-radios__input" id="q3--3" name="q3-" type="radio" value="Social media">
+                        <label class="govuk-label govuk-radios__label" for="q3--3">Social media</label>
+                      </div>
+                      <div class="govuk-radios__item">
+                        <input class="govuk-radios__input" id="q3--4" name="q3-" type="radio" value="I would not have used another contact method">
+                        <label class="govuk-label govuk-radios__label" for="q3--4">I would not have used another contact method</label>
+                      </div>
+                      <div class="govuk-radios__item">
+                        <input class="govuk-radios__input" id="q3-5" name="q3-" type="radio" value="Other" aria-controls="other-contact-details">
+                        <label class="govuk-label govuk-radios__label" for="q3--5">Other</label>
+                      </div>
+                      <div class="govuk-radios__conditional govuk-radios__conditional--hidden" id="other-contact-details">
+                        <div class="govuk-form-group">
+                          <label class="govuk-label" for="contact-by-other">Provide other contact options</label>
+                          <textarea class="govuk-textarea"  id="q4-" name="q4-" rows="5" aria-describedby="other-contact-details"></textarea>
+                        </div>
+                      </div>
+                    </div>
+                  </fieldset>
+                  <button id="submitPostChatSurvey" class="govuk-button">Submit</button>
+              </form>
+            </div>
+        </div>
+    </div>
+`
+
+export default class PostChatSurveyDigitalAssistant {
+    constructor(onSubmitted) {
+        this.onSubmitted = onSubmitted;
+    }
+
+    attachTo(container) {
+        this.container = container;
+
+        this.wrapper = document.createElement("div");
+        this.wrapper.id = "postChatSurveyWrapper";
+        this.wrapper.insertAdjacentHTML("beforeend", html);
+        container.appendChild(this.wrapper);
+
+        this.wrapper.querySelector("#submitPostChatSurvey").addEventListener(
+            "click",
+            (e) => {
+                this.onSubmitted(this);
+            }
+        );
+
+        $('input[name="q3-"]').on('click', function() {
+           if ($(this).val() != 'Other') {
+                document.getElementById("q4-").value = "";
+           }
+        });
+    }
+
+    detach() {
+        this.container.removeChild(this.wrapper)
+    }
+}
```

Comment:
Can we add New line at the end of the file please?

Reply:
Done